avm/ptn/lz/sub-vending/tests/e2e/hub-spoke/main.test.bicep

```
@@ -61,7 +72,7 @@ module testDeployment '../../../main.bicep' = {
     roleAssignments: [
       {
         principalId: '896b1162-be44-4b28-888a-d01acc1b4271'
-        definition: 'Network Contributor'
+        definition: '/providers/Microsoft.Authorization/roleDefinitions/4d97b98b-1d4f-4787-a291-c67834d212e7'
```

```suggestion
        definition: '/providers/Microsoft.Authorization/roleDefinitions/4d97b98b-1d4f-4787-a291-c67834d212e7' // 'Network Contributor'
```

The name is not supported anymore?